conf/github.yaml.template

```diff
@@ -0,0 +1,15 @@
+GITHUB:
```

Comment:
I'm unsure of the name here. github might be a bit too vague. can you think of something more specific for this config to be called?

Reply:
Changed to `github_repos`.